### PR TITLE
[resourcemonitor] Add update iptables filter cache

### PR DESCRIPTION
**Pull request by MykolaSolyanko**
Signed-off-by: Mykola Solianko <mykola_solianko@epam.com>

### CODE REVIEW

File: resourcemonitor/resourcemonitor.go [hunk 1]
```diff
@@ -372,6 +373,12 @@ func (monitor *ResourceMonitor) run(ctx context.Context) {
 
 		case <-monitor.pollTimer.C:
 			monitor.Lock()
+			if monitor.trafficMonitoring != nil {
+				if err := monitor.trafficMonitoring.UpdateIptablesFilterCache(); err != nil {
+					log.Error(err)
```
Comment (al1img) [inline resourcemonitor/resourcemonitor.go:378]: Add proper error message.

File: resourcemonitor/resourcemonitor.go [hunk 2]
```diff
@@ -72,6 +72,7 @@ type MonitoringSender interface {
 type TrafficMonitoring interface {
 	GetSystemTraffic() (inputTraffic, outputTraffic uint64, err error)
 	GetInstanceTraffic(instanceID string) (inputTraffic, outputTraffic uint64, err error)
+	UpdateIptablesFilterCache() error
```
Comment (al1img) [inline resourcemonitor/resourcemonitor.go:75]: Add more info in commit message why this API is needed and what it actually does.

**Review by Eabramov84 — APPROVED** [review]

File: resourcemonitor/resourcemonitor.go (hunk 1 above)
Comment (MykolaSolyanko) [reply resourcemonitor/resourcemonitor.go]: Done

File: resourcemonitor/resourcemonitor.go (hunk 2 above)
Comment (MykolaSolyanko) [reply resourcemonitor/resourcemonitor.go:75]: Add descriptions to commit message

Comment (al1img) [reply resourcemonitor/resourcemonitor.go:75]: I think we should not expose implementation details to the interface. Resource monitoring should not care about updating iptables filter cache. It should not know that monitoring is done using iptables and so on. Probably we can rename this method to something like `PrepareTrafficMonitoring` but a better solution is to do everything in the network manager.

**Pull request closed without merging**